hledger-lib/package.yaml

```diff
@@ -57,7 +57,7 @@ dependencies:
 - easytest
 - filepath
 - hashtables >=1.2.3.1
-- megaparsec >=6.4.1 && < 7
+- megaparsec >=6.4.1
```

Comment:
Huh.. I thought it would be megaparsec >=7. Do we support both 6 and 7 ?

Reply:
Ah, this is a mistake on my part. The code in this PR will only work with megaparsec 7, so the bounds should be megaparsec >= 7.0.0